scene/main/viewport.cpp

```diff
@@ -1222,6 +1222,21 @@ void Viewport::_propagate_viewport_notification(Node *p_node, int p_what) {
 	}
 }
 
+void Viewport::force_can_drop_data_check() {
```

Comment:
```suggestion
void Viewport::force_can_drop_data_check() {
	if (!gui.dragging) {
		return;
	}
```
Should have a check, in case it is not actually dragging.